Src/coffeescript/cql-execution/src/elm/query.coffee

```diff
@@ -28,6 +27,8 @@ module.exports.Without = class Without extends With
   exec: (ctx) ->
     !super(ctx)
 
+module.exports.Sort = class Sort extends UnimplementedExpression
+
```

Comment:
Whats the purpose of putting this back in?  Are we waiting to see if Bryns translator changes bring this up to the surface?